extra-multimedia/kodi/autobuild/beyond

```diff
@@ -0,0 +1 @@
+cd "$SRCDIR"
```

Comment:
This should not be necessary, there was a scripting error (unpaired `pushd`) in `prepare`.